src/gui/XournalView.cpp

```diff
@@ -451,9 +455,6 @@ void XournalView::scrollTo(size_t pageNo, double yDocument)
 	int height = v->getDisplayHeight();
 
 	layout->ensureRectIsVisible(x, y, width, height);
-
-	// Select the page
-	control->firePageSelected(pageNo);
```

Comment:
Restore this... we need to firePageSelected here! 
Page navigation won't work without it.